Add system agent images to package script

PennyScissors opened the pull request:
https://github.com/rancher/rancher/issues/32600

rancher-images.txt diff between master and this branch after running local drone builds with TAG=v2.6.0
```
--- ../rancher-images-master.txt        2021-08-19 22:39:52.000000000 -0700
+++ bin/rancher-images.txt      2021-08-19 23:00:02.000000000 -0700
@@ -203,6 +203,9 @@
 rancher/security-scan:v0.2.3
 rancher/shell:v0.1.10
 rancher/shell:v0.1.8
+rancher/system-agent-installer-k3s:v1.21.3-k3s1
+rancher/system-agent-installer-rke2:v1.21.3-rc6-rke2r2
+rancher/system-agent:v0.0.1-alpha37-suc
 rancher/system-upgrade-controller:v0.7.3
 rancher/system-upgrade-controller:v0.7.4
 rancher/tekton-utils:v0.1.2
```

paynejacob commented on Dockerfile.dapper:
```diff
@@ -80,6 +80,8 @@ RUN curl -sLf ${!DOCKER_URL} -o /usr/bin/docker && chmod +x /usr/bin/docker
 ENV YQ_URL=https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_${ARCH}
 RUN curl -sLf ${YQ_URL} -o /usr/bin/yq && chmod +x /usr/bin/yq
 
+RUN zypper install -y jq
```
why not add this to the install on line 18, we should at least put it higher up as it is less likely to change than the other dependencies.

PennyScissors replied:
Ok, I'll move it. I just put it there because is right after where we install yq.

kinarashah commented on scripts/package:
```diff
@@ -20,6 +20,22 @@ cp ../bin/rancher.yaml ../bin/rancher-namespace.yaml ../bin/rancher ../bin/agent
+# Replace + for - and construct an array of tags.
+RKE2_RELEASE_TAGS=( $(echo $RKE2_RELEASE_VERSIONS | tr + -) )
+# Prefix image repo to tags.
+SYSTEM_AGENT_INSTALLER_RKE2_IMAGES=( "${RKE2_RELEASE_TAGS[@]/#/rancher/system-agent-installer-rke2:}" )
```
Why are installer images not prefixed with `{REPO}/` like others but hardcoded with rancher/?

PennyScissors replied:
Good catch, let me update that.

PennyScissors commented:
Addressed comments and ran build again, diff still looks good:
```
--- ../rancher-images-master.txt        2021-08-19 22:39:52.000000000 -0700
+++ bin/rancher-images.txt      2021-08-20 09:41:40.000000000 -0700
@@ -203,6 +203,9 @@
 rancher/security-scan:v0.2.3
 rancher/shell:v0.1.10
 rancher/shell:v0.1.8
+rancher/system-agent-installer-k3s:v1.21.3-k3s1
+rancher/system-agent-installer-rke2:v1.21.3-rc6-rke2r2
+rancher/system-agent:v0.0.1-alpha37-suc
 rancher/system-upgrade-controller:v0.7.3
 rancher/system-upgrade-controller:v0.7.4
 rancher/tekton-utils:v0.1.2
 ```

dramich commented on Dockerfile.dapper:
```diff
@@ -15,7 +15,7 @@ ENV KUSTOMIZE_VERSION v3.10.0
 ENV CATTLE_KDM_BRANCH=dev-v2.6
 
-RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core wget curl tar vim less file xz gzip sed gawk iproute2 iptables
+RUN zypper -n install -y gcc binutils glibc-devel-static ca-certificates git-core wget curl tar vim less file xz gzip sed gawk iproute2 iptables jq
```
The -y isn't needed, -n already covers this https://www.mankier.com/8/zypper#Global_Options

PennyScissors replied:
Ah, I put the `-n` after the `install` when I tested this and it didn't work so I added the `-y` 🤦
